tests/test_meshing_utilities.py

```diff
@@ -697,9 +697,9 @@ def test_meshing_utilities(new_meshing_session):
 
     assert (
         meshing_session.meshing_utilities.get_maxsize_cell_zone_by_count(
-            zone_name_list=["outlet", "inlet", "wall", "internal"]
+            zone_name_list=["elbow-fluid"]
```

Comment:
👍 It is passing in PyFluent but raising an error in PyConsole. Not sure why this is failing in PyConsole. Thanks for updating this.

Reply:
Yes, it is strange. There is no exception in PyFluent, even in the Fluent console.

Reply:
What was the previous test doing instead of returning `None`?

Reply:
Nothing, just returning None.

Reply:
That's why the test was passing in CI.

Reply:
yes, actually the assertions were corrected in test as well.